client_sdk/avalon_client_sdk/ethereum/ethereum_worker_registry_list_client.py

```diff
@@ -160,7 +178,26 @@ def registry_lookup(self, app_type_id=None):
                 lookupResult = \
                     self.__contract_instance.functions.registryLookUp(b"") \
                     .call()
-            return lookupResult
+            wCount, lookUpString, wList = lookupResult
+            if lookup_next:
+                residue_list = []
+                for reg in wlist:
+                    if reg.orgId != lookup_tag:
```

Comment:
So is lookup_tag corresponds to org ID. It's not very obvious to follow the code. Comments will help